### PR TITLE
feat(action/gitTag): advanced parameter bool vprefix

### PR DESCRIPTION
**Pull request by yesnault**
if git.describe return a tag with a 'v', keep it to compute cds.semver

New advanced parameter on gitTag action, prefix 'v' is disabled by default
If false, this field is not exported

This will be usefull for go 1.11 and allow using the default value
of tag-version-prefix for node users:
https://docs.npmjs.com/misc/config#tag-version-prefix

Signed-off-by: Yvonnick Esnault <yvonnick@esnau.lt>

### CODE REVIEW

**Comment by fsamin** [comment]
advanced parameter should be a prefix (`v`or something else)

File: engine/api/action/builtin.go [hunk 1]
```diff
@@ -207,6 +207,13 @@ Semver used if fully compatible with https://semver.org/
 		Value:       "{{.cds.workspace}}",
 		Type:        sdk.StringParameter,
 	})
+	gittag.Parameter(sdk.Parameter{
+		Name:        "prefix",
+		Description: "Prefix for tag name",
+		Value:       "false",
```
Comment (bnjjj) [inline engine/api/action/builtin.go]: to change

**Comment by bnjjj** [comment]
cc @maxatome

File: docs/content/workflows/pipelines/actions/builtin/gittag.md [hunk 2]
```diff
@@ -18,6 +18,7 @@ This action creates a tag. You can use a pgp key to sign it.
 * tagMessage - optional - Message for the tag
 * path - optional - path to your git repository
 * signKey - optional - pgp key to sign the tag
+* Advanced parameter: vprefix - if checked, add 'v' prefix in tag created
```
Comment (bnjjj) [inline docs/content/workflows/pipelines/actions/builtin/gittag.md]: to change

**Comment by bnjjj** [comment]
CDS Report ut-engine#100.0 ✘  
* 
  * Engine Test with DB ✘

**Comment by ovh-cds** [comment]
CDS Report build-ui#5886.0 ✘  
* Compile
  * Build UI -  
  * TU All -  
  * TU model ✔ 
  * TU service ✘  
  * TU shared ✔ 
  * TU views ✔ 
Unit Tests Report
* HeadlessChrome 0.0.0 (Linux 0.0.0)












* HeadlessChrome 0.0.0 (Linux 0.0.0).1977656


































* HeadlessChrome 0.0.0 (Linux 0.0.0).1977654



  * CDS: application Store should add/update/delete a variable ✘
  * CDS: application Store should add/update/delete a permission ✘
  * CDS: pipeline Store Create and Delete Pipeline ✘

  * CDS: pipeline Store should create/update and delete a stage ✘
  * CDS: pipeline Store should create/update and delete a job ✘
  * CDS: pipeline Store should add/update/delete a permission ✘
  * CDS: pipeline Store should add/update/delete a parameter ✘
  * CDS: project Store should Get Projects ✘
  * CDS: project Store should create a Project ✘
  * CDS: project Store should update Project ✘

  * CDS: project Store should add/update/delete a variable ✘

  * CDS: project Store should add/update/delete an environment ✘


  * CDS: User Service &#43; Authent Store login ✘

* HeadlessChrome 0.0.0 (Linux 0.0.0).1977655

**Comment by yesnault** [comment]
@fsamin done

**Pull request merged**